config/reader.js

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
First of all I really love the idea of having one file that is responsible for reading all our config files. This is definitely way to go.

I have minor comment here, can we put this file in `/shared/` folder? In my opinion `/config/` folder should contain only `*.json` configuration files.